src/taipy/core/data/pickle.py

```diff
@@ -125,6 +127,8 @@ def _read(self):
         return pickle.load(open(self._path, "rb"))
 
     def _write(self, data):
+        if isinstance(data, (pd.DataFrame, pd.Series)):
```

Comment:
Should we set the env var when reading as well?

Reply:
I was thinking we don't have to since we don't what data is in the pickle file and if it is needed. But it does make sense that in the scenario where the data is a modin dataframe, we can safely unpickle it. I'll add the env in the read function 😄